### ISSUE TITLE
Extension is no longer on Firefox Add-ons

### ISSUE THREAD

**Issue by dkter** [SssWind/Learn-Darklight#5]
The [addon page](https://addons.mozilla.org/en-US/firefox/addon/learn-darklight/) for this extension seems to be gone.

**Comment by imashnake0** [comment]
F

**Comment by jakobdevey** [comment]
Yeah I just switched over to firefox and now I'm sad. Any idea if its coming back?

**Comment by dkter** [comment]
You can install it manually for now:
* Clone this repo
* Type `about:debugging` into the Firefox URL bar
* Go to "This Firefox" on the left
* Click "Load Temporary Add-on"
* Go to the Learn_DarkLight_Firefox folder and select any file (i.e. manifest.json)

**Comment by jakobdevey** [comment]
Awesome! Thank you so much!

**Comment by imashnake0** [comment]
Works like a charm, thank you @dkter!

**Comment by lucs100** [comment]
This worked, tysm @dkter :)
